### PR TITLE
Add naver-search extension

### PR DESCRIPTION
**Pull request by kw-lee**
## Description

naver-search extension
An extension for [Naver](https://naver.com) search with autosuggestions.
This extension is inspired by [alfnaversearch : Naver Search Workflow for Alfred](https://github.com/Kuniz/alfnaversearch).

## Screencast

search
shopping
endic
krdic

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/naver-search/src/search-api.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import fetch from "node-fetch";
+import { SearchType } from "./types";
+import { showToast, Toast } from "@raycast/api";
+
+export async function searchGeneralWords(wordToSearch: string, type: SearchType): Promise<string[][]> {
+  let url;
+  if (type == SearchType.GENERAL) {
+    url = `http://ac.search.naver.com/nx/ac?q_enc='utf-8'&q=${wordToSearch}&st=100&r_lt=100`;
```
Comment (pernielsentikaer) [inline extensions/naver-search/src/search-api.ts:8]:
```suggestion
    url = `https://ac.search.naver.com/nx/ac?q_enc='utf-8'&q=${wordToSearch}&st=100&r_lt=100`;
```

File: extensions/naver-search/src/search-dict-results.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    setLoading(true);
+    searchWords(search, type).then((words) => {
+      setWords(words);
+      setLoading(false);
+    });
+  }, []);
+
+  let baseurl: string;
+  if (type == SearchType.ENEN) {
+    baseurl = `http://endic.naver.com/search.nhn?ie=utf8&query=`;
```
Comment (pernielsentikaer) [inline extensions/naver-search/src/search-dict-results.tsx:20]:
```suggestion
    baseurl = `https://endic.naver.com/search.nhn?ie=utf8&query=`;
```

File: extensions/naver-search/src/search-dict-results.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  } else if (type == SearchType.KOKO) {
+    baseurl = `http://krdic.naver.com/search.nhn?kind=all&query=`;
```
Comment (pernielsentikaer) [inline extensions/naver-search/src/search-dict-results.tsx:22]:
```suggestion
    baseurl = `https://krdic.naver.com/search.nhn?kind=all&query=`;
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @kw-lee 👋 

Thanks for your contribution 💪 

I have some feedback ready for you to address:

- Could you move all screenshots to a folder called `media` in the root and update the README.

- The commands should have `"subtitle": "Naver",` but struggling a bit with matching titles, can you look into this?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by kw-lee** [comment]
Thanks for your feedback!
I reflect the feedback you sent, and add Chinese character dictionary.

**Review by pernielsentikaer — COMMENTED** [review]
Hi @kw-lee 👋 

Much better 🔥 It's shaping out really nice 🚀 

I have some comments for you:

- The “Search” command, can we add `word` to the title, or what does it search for?

- At least “Shopping” is not encoding spaces (as “Search” does)  could you check it out?

- What can I search for with "Krdic" "Handic" and "Endic" to get a match?

**Comment by fe9lix** [comment]
Hey guys, I'm closing this because of inactivity. Feel free to reopen later again and address the feedback.

**Pull request closed without merging**